addons/mrp_account/models/stock_move.py

```diff
@@ -1,29 +1,21 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import _, models
+from odoo import models
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
-    def _get_value_data(self,
-        forced_std_price=False,
-        at_date=False,
-        ignore_manual_update=False):
-        self.ensure_one()
-        if self.production_id:
-            valued_qty = self._get_valued_qty()
-            return {
-                'value': self._get_value_from_production(valued_qty),
-                'quantity': valued_qty,
-                'description': _('From Production Order %(reference)s', reference=self.production_id.name),
-            }
-        return super()._get_value_data(forced_std_price, at_date, ignore_manual_update)
-
-    def _get_value_from_production(self, quantity):
+    def _get_value_from_production(self, quantity, at_date=None):
         # TODO: Maybe move _cal_price here
         self.ensure_one()
-        return quantity * self.price_unit
+        if not self.production_id:
+            return super()._get_value_from_production(quantity, at_date)
+        return {
+            'value': quantity * self.price_unit,
+            'quantity': quantity,
+            'description': self.env._('Manufactured %(production)s', production=self.production_id.display_name),
```

Comment:
```suggestion
            'description': self.env._('From %(production)s', production=self.production_id.display_name),
```
Can't we simply put `From WH/MO/00008`?

Reply:
I will use the template than PO/INV